packages/cactus-plugin-consortium-manual/src/test/typescript/unit/consortium/get-node-jws-endpoint-v1.test.ts

```diff
@@ -75,16 +76,16 @@ test("Can provide JWS", async (t: Test) => {
   await pluginConsortiumManual.getOrCreateWebServices();
   await pluginConsortiumManual.registerWebServices(expressApp);
 
-  const epOpts: IGetNodeJwsEndpointOptions = {
+  /*  const epOpts: IGetNodeJwsEndpointOptions = {
     plugin: pluginConsortiumManual,
     consortiumRepo,
     keyPairPem,
-  };
+  };*/
   const pubKeyPem = keyPair.toPEM(false);
 
-  const ep = new GetNodeJwsEndpoint(epOpts);
+  // const ep = new GetNodeJwsEndpoint(epOpts);
```

Comment:
@AzaharaC It's cool to delete this if no longer needed, if we later figure that it is needed we can always take a peek at the git history to restore it so no need to keep a backup of it in the comments.